src/igvfd/schemas/changelogs/whole_organism.md

```diff
@@ -1,37 +1,40 @@
-## Changelog for *`whole_organism.json`*
+# Changelog for *`whole_organism.json`*
```

Comment:
why did you modify all the "#" ? I think you should stay consistent with all the other changelogs

Reply:
i was having issues with linting following rebase/resolution of conflicts, autopep8 wasn't passing so I had to run it on several files from the command line before it passed. This was one of the files.